### PR TITLE
updated client.rb file removed unnecessary line 

### PR DESCRIPTION
**Pull request by SabujGolui**
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Comment by akhilgkrishnan** [comment]
@abinashpa Please verify this

File: app/models/client.rb [hunk 1]
```diff
@@ -30,9 +30,7 @@
 class Client < ApplicationRecord
   include Discard::Model
   include UtilityFunctions
-
-  default_scope -> { kept }
-
+
```
Comment (abinashpa) [inline app/models/client.rb:33]: Can you fix the rubocop linting error?

Comment (abinashpa) [reply app/models/client.rb:33]: @SabujGolui Run `rubocop -A` to fix the linting issue.

**Comment by supriya3105** [comment]
@abinashpa Can you fix the issue and merge this PR today.

**Comment by abinashpa** [comment]
@akhilgkrishnan Can you please look into the failing test? It is failing to comment on PR which is very weird.

**Pull request merged**